lib/Pod/To/HTML.pm

```diff
@@ -10,26 +10,31 @@ sub colored($text, $how) {
     $text
 }
 
-multi method render($pod) {
-    pod2html($pod)
-}
-
-multi method render(Array $pod, Str :$header = '', Str :$footer = '', Str :head-fields($head) = '', :$default-title = '', :$lang = 'en') {
-    pod2html($pod, :$header, :$footer, :$head, :$default-title, :$lang)
-}
-
-multi method render(Pod::Block $pod, Str :$header = '', Str :$footer = '', Str :head-fields($head) = '', :$default-title = '', :$lang = 'en') {
-    pod2html($pod, :$header, :$footer, :$head, :$default-title, :$lang)
-}
-
-multi method render(IO::Path $file, Str :$header = '', Str :$footer = '', Str :head-fields($head) = '', :$default-title = '', :$lang = 'en') {
-    use MONKEY-SEE-NO-EVAL;
-    pod2html(EVAL($file.slurp ~ "\n\$=pod"), :$header, :$footer, :$head, :$default-title, :$lang);
-}
-
-multi method render(Str $pod-string, Str :$header = '', Str :$footer = '', Str :head-fields($head) = '', :$default-title = '', :$lang = 'en') {
-    use MONKEY-SEE-NO-EVAL;
-    pod2html(EVAL($pod-string ~ "\n\$=pod"), :$header, :$footer, :$head, :$default-title, :$lang);
+method render(
+    $pod,
+    Str :$header = '',
+    Str :$footer = '',
+    Str :head-fields($head) = '',
+    :$default-title = '',
+    :$lang = 'en'
+    ) {
+    given $pod {
+        when Array {
+            pod2html($pod, :$header, :$footer, :$head, :$default-title, :$lang)
+        }
+        when Pod::Block {
+            pod2html($pod, :$header, :$footer, :$head, :$default-title, :$lang)
+        }
+        when IO::Path {
+            use MONKEY-SEE-NO-EVAL;
+            pod2html(EVAL($file.slurp ~ "\n\$=pod"), :$header, :$footer, :$head, :$default-title, :$lang);
+        }
+        when Str {
+            use MONKEY-SEE-NO-EVAL;
+            pod2html(EVAL($pod-string ~ "\n\$=pod"), :$header, :$footer, :$head, :$default-title, :$lang);
```

Comment:
You still use `$file` and `$pod-string` here, despite changing the variable name to `$pod`. Please run `make test` on the changes.